docs/reference/modules/cluster/remote-clusters-api-key.asciidoc

```diff
@@ -0,0 +1,177 @@
+[[remote-clusters-api-key]]
+=== Add remote clusters using API key authentication
+
+coming::[8.10]
```

Comment:
Let's keep it for now. Folks sometimes get to these docs before the new version is release, which leads to confusion and frustration because this doesn't work on their older version. I've made a note to remove this on release day.